cmd/boltzcli/commands.go

```diff
@@ -1159,13 +1159,20 @@ func createSwap(ctx *cli.Context) error {
 	}
 
 	if externalPay {
-		printDeposit(amount, swap.Address, swap.TimeoutHours, uint64(swap.TimeoutBlockHeight), pairInfo.Limits)
-		fmt.Println()
+		printDeposit(swap.ExpectedAmount, swap.Address, swap.TimeoutHours, uint64(swap.TimeoutBlockHeight), pairInfo.Limits)
 	}
 
-	fmt.Println("Swap ID:", swap.Id)
-
-	return swapInfoStream(ctx, swap.Id, false)
+	if swap.Id == "" {
+		if !externalPay {
+			fmt.Println()
+			fmt.Println("Invoice has been paid directly via onchain:", swap.TxId)
```

Comment:
how should we word this?

Reply:
How about "Invoice has been paid directly to the wallet: <txId>"